DOL.WHD.Section14c.Web/src/modules/components/sectionWorkSites/sectionWorkSitesController.test.js

```diff
@@ -60,6 +60,20 @@ describe('sectionWorkSitesController', function() {
     expect(controller.activeWorkerIndex).toBe(-1);
   });
 
+  it('cancel and close pane clears the active worker', function() {
+    var e = jasmine.createSpyObj('e', ['preventDefault']);
+
+    var controller = sectionWorkSitesController();
+    spyOn(controller, 'clearActiveWorker');
+    spyOn(controller, 'addEmployee');
+    controller.cancelAndClosePanel(e);
+
+    expect(controller.activeWorker).toEqual({});
+    expect(controller.activeWorkerIndex).toBe(-1);
+    expect(controller.clearActiveWorker).toHaveBeenCalled();
+    expect(controller.addEmployee).not.toHaveBeenCalled();
```

Comment:
Is it possible to spy on `closeSlidingPanel` so this test can verify that's called?  It's not attached to the controller/vm directly, so it might not be possible to reference it from here.  Maybe you could add a `this.closeSlidingPanel = closeSlidingPanel;` line to the controller code so that it's accessible from outside the controller?  Closing the panel is an important part of this function, so we should make sure it's happening.  😄